pkg/errors/definitions.go

```diff
@@ -169,17 +168,46 @@ nextArg:
 
 	def.setGRPCStatus() // store the (marshaled) gRPC status message.
 
-	Definitions[fullName] = &def
+	definitionsMu.Lock()
```

Comment:
I don't see why you wouldn't `defer` here. You want to avoid holding the lock while defining the `i18n` definition ?